.gitignore

```diff
@@ -64,7 +64,7 @@ libs/
 *.suo
 
 # Output
-output/*.csv
+output/tracking*.csv
```

Comment:
The problem with this as way to address #734 is that running an unmodified model will produce modified (from git's point of view) files, because of the timestamp at the top of the outputs...which I don't think is the behavior we want. It seems better to create an `example_outputs/` directory, perhaps as a subdirectory of `output/`.

Reply:
Hmmm that is a good point, do we want to add that or does it make sense to include the output streams as part of the release materials?